core/liblwm2m.h

```diff
@@ -497,13 +498,22 @@ typedef struct _lwm2m_observed_
     lwm2m_watcher_t * watcherList;
 } lwm2m_observed_t;
 
+typedef enum {
+    NOT_BOOTSTRAPPED = 0,
+    BOOTSTRAP_REQUESTED,
+    BOOTSTRAP_CLIENT_HOLD_OFF,
+    BOOTSTRAP_PENDING,
+    BOOTSTRAP_FINISHED,
```

Comment:
Just with "no message received for COAP_DEFAULT_MAX_AGE" (it's mostly still pascal's implementation).